# Hypothesis Testing

In [1]:
import math
from scipy.stats import norm

In [2]:
sample_mean = 3050
theoretical_mean = 1000 + 5 * 600  # W = 1000 + 5 * X, X = 600
sigma = 5 * 25  # σ = 5 * 25 units
n = 25  # sample size
alpha = 0.05  # significance level

In [3]:
# Calculate the test statistic
standard_error = sigma / math.sqrt(n)
t_statistic = (sample_mean - theoretical_mean) / standard_error

In [4]:
# Determine the critical value from the standard normal distribution table
critical_value = norm.ppf(1 - alpha)

In [5]:
if t_statistic > critical_value:
    decision = "Reject the null hypothesis"
else:
    decision = "Do not reject the null hypothesis"

In [6]:
print(f"Sample Mean: {sample_mean}")
print(f"Theoretical Mean: {theoretical_mean}")
print(f"Standard Error: {standard_error}")
print(f"Test Statistic (t): {t_statistic}")
print(f"Critical Value (Z): {critical_value}")
print(f"Decision: {decision}")

Sample Mean: 3050
Theoretical Mean: 4000
Standard Error: 25.0
Test Statistic (t): -38.0
Critical Value (Z): 1.6448536269514722
Decision: Do not reject the null hypothesis


# Conclusion:
## Based on the hypothesis test conducted:

#### The calculated test statistic was t=−38, which is significantly lower than the critical value of Z=1.645 at a 5% significance level.

#### Since the test statistic does not fall within the critical region (i.e., it is not greater than the critical value), we do not have sufficient evidence to reject the null hypothesis.


#### Conclusion: There is not strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the theoretical model suggests. The observed mean weekly cost of Rs. 3050 does not significantly deviate from the expected cost based on the model W=1000+5X for X=600 units. Therefore, it is reasonable to conclude that the theoretical cost model is still valid.